charts/nutanix-csi-storage/templates/ntnx-sc.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if eq .Values.volumeClass true }}
 
-{{- $storageContainer := .Values.storageContainer | required "storageContainer is required." -}}
+{{ $storageContainer := .Values.storageContainer | required "storageContainer is required." }}
 ---
```

Comment:
We have to add secret parameters like it use to be in 2.6.x for nutanix-volumes
https://github.com/nutanix/helm/blob/master/charts/nutanix-csi-storage/templates/ntnx-sc.yaml#L23

Reply:
we should also run some manual tests to verify helm changes. @koushikmalga